apps/console/src/features/users/pages/users.tsx

```diff
@@ -139,13 +139,25 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
                 setUsersList(moderateUsersList(data, modifiedLimit, TEMP_RESOURCE_LIST_ITEM_LIMIT_OFFSET));
                 setUserStoreError(false);
             }).catch((error) => {
+                if (error?.response?.data?.description) {
```

Comment:
Shall we check if the `response.data.detail` as well here. Check the API swagger and verify if the error message is coming under `description` or `detail`